target_athena/utils.py

```diff
@@ -57,140 +55,12 @@ def flatten_record(d, parent_key=[], sep="__"):
     return dict(items)
 
 
-def get_target_key(stream_name, prefix=None, timestamp=None, naming_convention=None):
+def get_target_key(stream_name, object_format, prefix="", timestamp=None, naming_convention=None):
```

Comment:
Rather than later appending `.gz` to this key, what do you think of adding a `compression` arg here and factoring it into the naming here in this function?